modules/combined/tests/elastic_thermal_patch/elastic_thermal_jacobian_rz_smp_test.i

```diff
@@ -164,88 +150,81 @@
     boundary = 10
     function = temp
   [../]
-
-[] # BCs
+[]
 
 [Materials]
-
-  [./stiffStuff1]
-    type = Elastic
-    block = 1
-
-    disp_r = disp_x
-    disp_z = disp_y
-
+  [./elasticity_tensor]
+    type = ComputeIsotropicElasticityTensor
     youngs_modulus = 1e6
     poissons_ratio = 0.25
+  [../]
 
-    temp = temp
-    thermal_expansion = 1e-6
+  [./strain]
+    type = ComputeAxisymmetricRZIncrementalStrain
+    displacements = 'disp_x disp_y'
+    eigenstrain_names = eigenstrain
+  [../]
+
+  [./thermal_strain]
+    type = ComputeThermalExpansionEigenstrain
+    temperature = temp
+    stress_free_temperature = 117.56
+    thermal_expansion_coeff = 1e-6
+    incremental_form = true
+    eigenstrain_name = eigenstrain
+  [../]
+
+  [./stress]
+    type = ComputeStrainIncrementBasedStress
   [../]
 
   [./heat]
     type = HeatConductionMaterial
-    block = 1
-
     specific_heat = 0.116
     thermal_conductivity = 4.85e-4
   [../]
 
   [./density]
     type = Density
-    block = 1
     density = 0.283
     disp_r = disp_x
     disp_z = disp_y
   [../]
-
-[] # Materials
+[]
 
 [Preconditioning]
-#  [./FDP]
-#    type = FDP
-#    full = true
-#  [../]
   [./SMP]
     type = SMP
     full = true
   [../]
 []
 
 [Executioner]
-
   type = Transient
 
   #Preconditioned JFNK (default)
   solve_type = 'PJFNK'
 
-
-
   petsc_options_iname = '-pc_type -ksp_gmres_restart'
   petsc_options_value = 'lu       101'
 
-
   line_search = 'none'
 
-
   nl_abs_tol = 1e-9
   nl_rel_tol = 1e-12
-
-
   l_max_its = 20
-
   start_time = 0.0
   dt = 1.0
   num_steps = 1
   end_time = 1.0
-[] # Executioner
+[]
 
 [Outputs]
-  file_base = out_jac_rz_smp
+  file_base = elastic_thermal_jacobian_rz_smp_out
```

Comment:
We are developing solid_mechanics and tensor_mechanics tests that we want to be able to compare against the same gold file.  We're naming the solid_mechanics one with a '_sm' extension, and having them write to the same output file, using prereq to make sure they don't both write to it at once. I think we need to at least put the file_base parameter in the _sm version of the test, don't we?